CHANGELOG

```diff
@@ -13,6 +13,17 @@ The rules for this file:
   * accompany each entry with github issue/PR number (Issue #xyz)
 
 ------------------------------------------------------------------------------
+12/12/2025 Pradyumn-cloud
+  * 1.1.0
+
+  Changes
+
```

Comment:
All of your changes should go into the existing 1.1.0 section below. Leave the date as ??/??/???? and just
- add your handle after orbeckst
- add the one line "Added MRC file writing support (Issue #108)" in the section *Enhancements*